File: atv_academia/NatalFitness.java
public class NatalFitness {
    BancoDeDados db;


    public NatalFitness(BancoDeDados db){
        this.db = db;
    }

    public void addFuncionario(Funcionario funcionario){
        this.db.addFuncionarios(funcionario);
    }

    public void imprimirFuncionarios() {
        this.db.imprimirFuncionarios();
    }
    public void imprimirProfessores() {
        this.db.imprimirProfessores();
    }
    public void imprimirRecepcionistas() {
        this.db.imprimirRecepcionistas();
    }
}

File: atv_academia_kotlin/src/main/kotlin/NatalFitness.kt
class NatalFitness(var db: BancoDeDados) {
    fun addFuncionario(funcionario: Funcionario?) {
        db.addFuncionarios(funcionario!!)
    }

    fun imprimirFuncionarios() {
        db.imprimirFuncionarios()
    }

    fun imprimirProfessores() {
        db.imprimirProfessores()
    }

    fun imprimirRecepcionistas() {
        db.imprimirRecepcionistas()
    }
}
